ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -60,7 +69,12 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
           var $editText = $(React.findDOMNode(this.refs.editText));
           $editText.focus();
         });
-      } else if (formatActionState === 'edit') {
+      }
+    },
+    saveTerm: function() {
+      var formatActionState = this.state.formatActionState;
+
+      if (formatActionState === 'edit') {
```

Comment:
It looks like you can remove this line too since the button using `saveTerm` is only available when this condition is true